index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+  <link rel="stylesheet" href="./styles.css">
+</head>
+<body>
+  
+  <header>
+    <nav>
+      <div class="nav-bar">
+        <a href="#" class="logo">Kajal</a>
+        <button class="hambug-menu">
+         <a href="#"><img src="./img/menu.svg" alt="menu"></a>
+        </button>
+      </div>
+    </nav>
+
+    <section class="head-section">
+      
+      <h1 class="header-title">I'm Kajal Pramanik to see you!</h1>
```

Comment:
[OPTIONAL]
- [ ] Please consider updating the color of the header text to make it match Figma design

## Your design:
![image](https://user-images.githubusercontent.com/50721479/191824885-d0c05e12-96b3-4c2a-bdf1-9c06dc5bae2f.png)

## Figma design:
![image](https://user-images.githubusercontent.com/50721479/191824866-6f0487ac-cbda-4d9f-b987-1b8d2f7dc5ef.png)